utils/awsFunctions.ts

```diff
@@ -0,0 +1,136 @@
+import {
+    IdentitystoreClient,
+    ListUsersCommand,
+    CreateUserCommand,
+    CreateGroupMembershipCommand,
+    ListUsersCommandInput,
+    CreateUserCommandInput,
+    CreateGroupMembershipCommandInput,
+  } from "@aws-sdk/client-identitystore";
+  import config from "config";
+  
+  // Define the configuration variables with proper types
+  const accessKeyId: string = config.get<string>("aws.access_key");
+  const secretAccessKey: string = config.get<string>("aws.secret_key");
+  const region: string = config.get<string>("aws.region");
+  const identityStoreId: string = config.get<string>("aws.identity_store_id");
+  
+  let client: IdentitystoreClient;
+  
+  /**
+   * Configures AWS SDK credentials and returns a singleton IdentitystoreClient instance.
+   * @returns {IdentitystoreClient} Singleton AWS Identitystore client
+   */
+  function configureAWSCredentials(): IdentitystoreClient {
```

Comment:
Does not it throw error for wrong keys?

Reply:
It will throw, but as we will be setting this at the time of deployment, so there won't be chances of inputting wrong keys

Reply:
If by any chance, the keys are not correct then how would we know it?

Reply:
It will throw connection error, don't remember the exact response but we can understand from the response returned